src/snowflake/snowpark/modin/plugin/docstrings/series.py

```diff
@@ -3659,6 +3659,38 @@ def hasnans():
         Return True if there are any NaNs.
         """
 
+    @property
+    def is_monotonic_increasing():
+        """
```

Comment:
add "Returns" section in both docstrings

Reply:
you also have the names for both switched

Reply:
Good catch, thank you!